Module-3-Javascript-Coding-Task/src/palindrome.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+// Check if the text is a palindrome. For empty string, it is not considered a palindrome. Whitespaces should not be counted
+
+// solution #1
+
+export const isPalindrome = (text) => !text ? false : text.replace(/ /g, '').toLowerCase().split('').reverse().join('').replace(/ /g, '') === text.toLowerCase().replace(/ /g, '')
```

Comment:
second `replace(/ /g, '')` is not needed after the join

Reply:
corrected